.ci/acceptance-tests/inspec-integration.sh

```diff
@@ -29,44 +23,23 @@ for i in $(find products/ -name 'inspec.yaml' -printf '%h\n');
 do
   bundle exec compiler -p $i -e inspec -o "build/inspec/"
 done
-pushd build/inspec/test/integration
+pushd build/inspec
 
-# Generate tfvars
-pushd attributes
-ruby compile_vars.rb > terraform.tfvars
-mv terraform.tfvars ../terraform
-popd
-
-# Run terraform
-pushd terraform
-terraform init
-terraform plan
-
-export TF_PATH=${PWD}
-trap cleanup EXIT
-terraform apply -auto-approve
-export GOOGLE_APPLICATION_CREDENTIALS="${PWD}/inspec.json"
-popd
+# Setup for using current GCP resources
+export GCP_PROJECT_NUMBER=542134042613
```

Comment:
I'd prefer if we don't hardcode project name + number in here and use credhub instead.